src/api/spec/jobs/consistency_check_job_spec.rb

```diff
@@ -28,6 +28,7 @@
     let!(:project) { create(:project, name: 'super_project') }
     let(:consistency_checkjob) { described_class.new }
     let(:error_message) { "Project meta is different in backend for super_project\n{:foo=>\"bar\"}" }
+    let!(:admin_user) { create(:admin_user, login: 'Admin') }
```

Comment:
@hennevogel why do you had to add this now? The logic of the code is stayed the same, just curious

Reply:
We truncate the users table between test runs, including the Admin user.